python-sdk/src/astro/sql/operators/dataframe.py

```diff
@@ -215,6 +225,48 @@ def _convert_column_capitalization_for_output(function_output, columns_names_cap
             )
         return function_output
 
+    def get_openlineage_facets(self, task_instance) -> OpenLineageFacets:  # skipcq: PYL-W0613
+        """
+        Collect the input, output, job and run facets for DataframeOperator
+        """
+        input_dataset: list[OpenlineageDataset] = []
+
+        output_dataset: list[OpenlineageDataset] = [OpenlineageDataset(namespace=None, name=None, facets={})]
```

Comment:
can we initialise even `output_dataset` the same way as `input_dataset`?

Reply:
@rajaths010494 this is done. Please check now